src/eventreporter/EventData.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import MeetingHistoryState from '../eventcontroller/MeetingHistoryState';
+
+/**
+ * [[EventData]] describes event information types.
+ */
+export default interface EventData {
+  name: MeetingHistoryState;
```

Comment:
Can we document what these properties mean? And maybe give an example of how it is supposed to look?

Reply:
Yes, updated with few examples and added doc for each property in this interface. Thanks for the catch!